app/search_builders/hyrax/admin_set_search_builder.rb

```diff
@@ -8,6 +8,7 @@ class AdminSetSearchBuilder < ::SearchBuilder
     # @param [Symbol] access one of :edit, :read, or :deposit
     def initialize(context, access)
       @access = access
+      @rows = 100
```

Comment:
Is it possible to scope this change to `AdminSetService#search_results_with_work_count` ?